src/api.js

```diff
@@ -39,6 +39,13 @@ export default class BounceClient {
         return await fetch(this.url + endpoint, requestData);
     }
 
+    /**
+     * Set token to null
+     */
+    handleLogOut() {
```

Comment:
Good idea implementing this, but I think a `signOut` already exists in this file. You'll see it if you rebase from `master`.

Here it is: https://github.com/ubclaunchpad/bounce-front-end/blob/master/src/api.js#L52